Package.swift

```diff
@@ -237,6 +237,7 @@ let package = Package(
             name: "swiftpm-xctest-helper",
             dependencies: [],
             linkerSettings: [
+                .linkedFramework("XCTest", .when(platforms: [.iOS, .macOS, .tvOS, .watchOS])),
```

Comment:
should SwiftPM do it automatically for test targets?

Reply:
It does, but this isn't a test target.

Reply:
We do that, but this is an executable.

Reply:
oh boy

Reply:
It seems reasonable, conceptually, to ask for it, given that it's a specialized tools framework and not a standard system framework.  When I tried the same fix originally, I couldn't get it to work, however.  Is there a way of testing this, or will it be implicitly tested because you couldn't build SwiftPM without it once there is a new enough version on the CI?